src/Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -47,4 +47,4 @@
     {% endapply %}
 {% endblock %}
 
-{% block label %}{% if is_link is defined and is_link %}{{ icon|default|raw }}{% endif %}{% if options.allow_safe_labels and item.extra('safe_label', false) %}{{ item.label|raw }}{% else %}{{ item.label|trans({}, translation_domain|default('messages')) }}{% endif %}{% endblock %}
+{% block label %}{% if is_link is defined and is_link %}{{ icon|default|raw }}{% endif %}{% if options.allow_safe_labels and item.extra('safe_label', false) %}{{ item.getLabel()|raw }}{% else %}{{ item.getLabel()|trans({}, translation_domain|default('messages')) }}{% endif %}{% endblock %}
```

Comment:
```suggestion
# We use method accessor instead of ".label" since `item` implements `ArrayAccess` and could have a property called "label".
{% block label %}{% if is_link is defined and is_link %}{{ icon|default|raw }}{% endif %}{% if options.allow_safe_labels and item.extra('safe_label', false) %}{{ item.getLabel()|raw }}{% else %}{{ item.getLabel()|trans({}, translation_domain|default('messages')) }}{% endif %}{% endblock %}
```